python/lsst/ts/mtdome/mtdome_csc.py

```diff
@@ -261,7 +289,10 @@ async def connect(self) -> None:
         # DM-35794: Also send enabled event for Aperture Shutter.
         await self.evt_azEnabled.set_write(state=EnabledState.ENABLED)
         await self.evt_elEnabled.set_write(state=EnabledState.ENABLED)
-        await self.evt_shutterEnabled.set_write(state=EnabledState.ENABLED)
+        # Check supported event to make sure of backward compatibility with
+        # XML 12.0.
+        if support_dds_item("evt_shutterEnabled"):
```

Comment:
I think this method is only useful for commands because you need to make the introspection before calling `init` of the parent class.. but at this point you could simply do:

```
if hasattr(self, "evt_shutterEnabled"):
```

which won't require you to parse the idl file.

Reply:
Good tip, thanks! Done.